packages/settings/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "lib",
+    "baseUrl": ".",
+    "esModuleInterop": true
```

Comment:
- why do we need esModuleInterop?
- If we need it should it be on the base root tsconfig.json?

Reply:
For the `deep-freeze`

Reply:
I got some compilation errors in other places when I tried to move it to the base tsconfig.json, I'll try to look into it soon